tensorflow_recommenders_addons/dynamic_embedding/core/BUILD

```diff
@@ -32,6 +32,25 @@ custom_op_library(
     deps = ["//tensorflow_recommenders_addons/dynamic_embedding/core/lib/cuckoo:cuckoohash"],
 )
 
+custom_op_library(
+    name = "_redis_table_ops.so",
+    srcs = [
+        "kernels/redis_impl/md5.h",
+        "kernels/redis_impl/md5.cc",
+        "kernels/redis_impl/redis_connection_util.hpp",
+        "kernels/redis_impl/redis_connection_pool.hpp",
+        "kernels/redis_impl/redis_cluster_connection_pool.hpp",
+        "kernels/redis_table_op.h",
+        "kernels/redis_table_op.cc",
+        "ops/redis_table_ops.cc",
+    ]   ,
+    deps = [
+        "@hiredis//:hiredis",
+        "@redis-plus-plus//:redis-plus-plus",
+    ]   ,
+    copts = ["-pthread", "-O3", "-mavx2", "-mavx512f", "-mfma", "-funroll-loops"],
```

Comment:
You can add to `custom_op_library` directly , but still need to consider some user environment without avx512 that will cause core dump.

Reply:
Already changed to "-march=native"